Releases.md

```diff
@@ -0,0 +1,34 @@
+# Releases
+
+*We are currently undergoing a project rename and repo transfer. Please bare with use as we work to improve the release process.*
+
+## nebari
+
+The release process for `nebari` is managed by a checklist which maintainers can create from a new issue template. When the time comes, that issue is used to track all related tasks and as a place to discuss the release. 
+
+> [Click here to create this release checklist issue.](https://github.com/Quansight/qhub/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=release-checklist.md&title=%5BRELEASE%5D+-+%3Cversion%3E)
```

Comment:
This link should render correctly once https://github.com/Quansight/qhub/pull/1314 is merged.